src/inspector/node_string.h

```diff
@@ -73,7 +73,7 @@ extern size_t kNotFound;
 }  // namespace inspector
 }  // namespace node
 
-#define DCHECK CHECK
-#define DCHECK_LT CHECK_LT
-
+#ifndef DCHECK
+  #define DCHECK CHECK
+  #define DCHECK_LT CHECK_LT
```

Comment:
There is a `endif` missing:

```suggestion
  #define DCHECK_LT CHECK_LT
#endif  // DCHECK
```